crates/cairo-coverage/src/input/statements_map.rs

```diff
@@ -0,0 +1,71 @@
+use crate::input::data_loader::types::{
+    CodeLocation, CoverageAnnotations, LineRange, ProfilerAnnotations,
+};
+use crate::types::{FileLocation, FunctionName};
+use anyhow::{Context, Result};
+use cairo_lang_sierra::debug_info::Annotations;
+use cairo_lang_sierra::program::StatementIdx;
+use serde::de::DeserializeOwned;
+use std::collections::HashMap;
+
+const PROFILER_NAMESPACE: &str = "github.com/software-mansion/cairo-profiler";
+const COVERAGE_NAMESPACE: &str = "github.com/software-mansion/cairo-coverage";
```

Comment:
Would be better to implement those as methods of a trait `Namespace` with method `key`(names tbd) and implement those for `CoverageAnnotations` and `ProfilerAnnotations` as those are tied. It is ok to leave as it is, just knowledge sharing.